Dockerfile

```diff
@@ -1,21 +1,32 @@
-FROM node:20-alpine
-# Some tools required when using alpine -> https://github.com/nodejs/docker-node/issues/282#issue-193774074
+FROM node:20-alpine as base
+
+WORKDIR /app
+
+
+FROM base as builder
+# Some tools required when using alpine -> https://github.com/nod# ejs/docker-node/issues/282#issue-193774074
 RUN apk add --no-cache --virtual .gyp python3 make g++
 
 ARG NODE_AUTH_TOKEN=""
 
-WORKDIR /app
-
 COPY package.json yarn.lock ./
 
-RUN yarn install --production
+RUN yarn install
```

Comment:
`--immutable` would make sense tho, right?

Reply:
true, that should still work